src/main/kotlin/dev/stalla/util/ItunesCategoryExtensions.kt

```diff
@@ -0,0 +1,125 @@
+package dev.stalla.util
+
+import dev.stalla.model.itunes.ItunesCategory
+
+@InternalApi
+internal val itunesCategoryValues: List<ItunesCategory> = listOf(
+    ItunesCategory.Arts,
```

Comment:
This list is the major drawback of this approach. Unlike with the `.values()` method of enums, we don't get access to all defined instances of the sealed class, unless we want to use reflection (which I don't). So we have to maintain a list ourselves. The threat here is that we forgot something. There are tests to ensure completeness.

Reply:
The lazy cache is a good idea!